### PR TITLE
agent: Add lctl action plugin

### PR DESCRIPTION
**Pull request by utopiabound**
lctl(Vec<String>) -> String

Fixes Issue #1378 

Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

### CODE REVIEW

File: iml-agent/src/action_plugins/lctl.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{
+    agent_error::ImlAgentError,
+    cmd::{self},
```
Comment (jgrund) [inline iml-agent/src/action_plugins/lctl.rs]:
```suggestion
    cmd,
```

Comment (utopiabound) [reply iml-agent/src/action_plugins/lctl.rs]: Don't I need cmd::{self} if I reference cmd?

Comment (jgrund) [reply iml-agent/src/action_plugins/lctl.rs]: It should work without it.

Comment (jgrund) [reply iml-agent/src/action_plugins/lctl.rs]: `self` is just a helper if you are importing other things under that namespace and still want to use `cmd` as well.

**Pull request merged**